src/core/server/elasticsearch/elasticsearch_service.test.ts

```diff
@@ -365,7 +361,6 @@ describe('#stop', () => {
       await elasticsearchService.stop();
       await delay(100);
       expect(mockedClient.nodes.info).toHaveBeenCalledTimes(1);
-      done();
```

Comment:
The callback here ensured we waited for the first event before completing the test. So to keep the same behavior we need to await the first event to ensure they are completed at the time of the test.

You can convert the subscription to a promise like this:
```
const promise = setupContract.esNodesCompatibility$.pipe(first()).toPromise();
await promise;
```